public/app/percona/shared/services/pmmDump/pmmDump.types.ts

```diff
@@ -0,0 +1,40 @@
+export interface PmmDumpResponse {
+  dumps: PmmDump[];
+}
+
+export interface PmmDump {
+  dump_id: string;
+  status: string;
+  node_ids: string[];
+  start_time: string;
+  end_time: string;
+  created_at: string;
+}
+
+export interface DeleteDump {
+  dump_ids: string[];
+}
+
+export interface Node {
+  node_id: string;
```

Comment:
If possible (or it's already implemented somewhere) use camelCase for objects from API (convert when recieved/sending).

Reply:
Updated.